api/add.go

```diff
@@ -13,10 +15,10 @@ var DefaultShardSize = uint64(100 * 1024 * 1024) // 100 MB
 // AddedOutput carries information for displaying the standard ipfs output
 // indicating a node of a file has been added.
 type AddedOutput struct {
-	Name  string `json:"name"`
-	Cid   string `json:"cid,omitempty"`
-	Bytes uint64 `json:"bytes,omitempty"`
-	Size  uint64 `json:"size,omitempty"`
+	Name  string  `json:"name" codec:"n,omitempty"`
+	Cid   cid.Cid `json:"cid,omitempty" codec:"c"`
```

Comment:
Does `omitempty` work for struct fields?

Reply:
removed `omitempty`